net.wz2100.wz2100.yaml

```diff
@@ -0,0 +1,159 @@
+      
+    sources:
+    - type: archive
+      url: 'http://download.qt.io/official_releases/qt/5.11/5.11.0/submodules/qtscript-everywhere-src-5.11.0.tar.xz'
+      sha256: 25926ae733b31baac7af51f489ad26570d4f4f02ad4892a4a82babae5f5168c5
+
+  - name: sdl2
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+    - -DCMAKE_INSTALL_PREFIX=/app
+    - -DSDL_STATIC=OFF
+    - -DSDL_DLOPEN=ON
+    - -DARTS=OFF
+    - -DESD=OFF
+    - -DNAS=OFF
+    - -DALSA=ON
+    - -DPULSEAUDIO_SHARED=ON
+    - -DVIDEO_WAYLAND=ON
+    - -DRPATH=OFF
+    - -DCLOCK_GETTIME=ON
+    sources:
+      - type: archive
+        url: 'https://www.libsdl.org/release/SDL2-2.0.8.tar.gz'
+        sha512: 673c6058b8692a36b4a3594456b10ef6051efe79e4fb644421fc5c76b11fd68b895840a2c8b72413418c378733e2993d33f19767d0d7ed101eda6310bd70c869
+  
+  - name: libogg
+    config-opts:
+    - --prefix=/app
+    - -disable-static
+    sources:
+      - type: archive
+        url: 'https://downloads.xiph.org/releases/ogg/libogg-1.3.3.tar.xz'
+        sha256: 4f3fc6178a533d392064f14776b23c397ed4b9f48f5de297aba73b643f955c08
+        
+  - name: libvorbis
+    config-opts:
+    - --prefix=/app
+    sources:
+      - type: archive
+        url: 'https://downloads.xiph.org/releases/vorbis/libvorbis-1.3.6.tar.gz'
+        sha256: 6ed40e0241089a42c48604dc00e362beee00036af2d8b3f46338031c9e0351cb
+        
+  - name: physfs
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+    - -DCMAKE_INSTALL_PREFIX=/app
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DPHYSFS_BUILD_TEST=OFF
+    sources:
+      - type: archive
+        url: 'http://icculus.org/physfs/downloads/physfs-3.0.1.tar.bz2'
+        sha1: 9959a7acad0aa30d1a86d3a418e61155b0ac533c
+      
+  - name: ffmpeg
+    config-opts:
+      - --disable-everything
+      - --enable-gpl
+      - --disable-static
+      - --enable-shared
+      - --disable-doc
+      - --disable-ffplay
+      - --disable-ffprobe
+    post-install:
+      - chrpath -d /app/bin/ffmpeg
+    cleanup:
+      - /share/ffmpeg/examples
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-4.0.2.tar.xz
+        sha256: a95c0cc9eb990e94031d2183f2e6e444cc61c99f6f182d1575c433d62afb2f97
+        
+  - name: openal
+    buildsystem: cmake-ninja
+    config-opts:
+    - -DCMAKE_INSTALL_PREFIX=/app
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DCMAKE_INSTALL_LIBDIR=lib
+    sources:
+      - type: git
+        url: https://github.com/kcat/openal-soft
+        tag: openal-soft-1.18.2
+        commit: ce6076091bac3c00cd10803916e8911495580bd0
+        
+  - name: fribidi
+    buildsystem: meson
+    config-opts:
+    - -Ddocs=false
+    sources:
+      - type: git
+        url: https://github.com/fribidi/fribidi
+        tag: v1.0.5
+        
+  - name: ttf-dejavu
+    buildsystem: simple
+    build-commands:
+      - install -dm755 /app/usr/share/fonts/TTF
+      - install -m644 ttf/*.ttf /app/usr/share/fonts/TTF/
```

Comment:
do you need to regenerate the fontconfig cache here to speed up app launch - sort of like https://github.com/flathub/com.github.ojubaorg.Othman/blob/master/com.github.ojubaorg.Othman.json#L40